tests/MongoDB.Driver.Tests/MongoCollectionImplTests.cs

```diff
@@ -1421,7 +1422,7 @@ private class ClassForDistinctWithArrayField
             var weights = new BsonDocument("y", 1);
             var storageEngine = new BsonDocument("awesome", true);
             var maxTime = milliseconds != null ? TimeSpan.FromMilliseconds(milliseconds.Value) : (TimeSpan?)null;
-            var createOneIndexOptions = new CreateOneIndexOptions { MaxTime = maxTime };
+            var createOneIndexOptions = usingCreateOneIndexOptions ? new CreateOneIndexOptions { MaxTime = maxTime } : null;
```

Comment:
Passing an explicit null for createOneIndexOptions effectively tests the scenario of omitting the argument (which just results in the default value of null being passed).